docs/externalGrading.md

```diff
@@ -163,15 +163,17 @@ The following is an example of a well-structured course layout:
 
 ## Grading results
 
-Your grading process must write its results to `/grade/results/results.json`. The result only has 2 mandatory fields: `succeeded` and `score`. `succeeded` indicates if the tests were able to run successfully, or if they failed due to some error. `score` is the score, and should be a floating point number in the range [0.0, 1.0].
+Your grading process must write its results to `/grade/results/results.json`. The result only has one mandatory field: `score`, which is the score for the submitted attempt, and should be a floating point number in the range [0.0, 1.0].
 
-As long as those two fields are present, you may add any additional data to that object that you want. This could include information like detailed test results, stdout/stderr, compiler errors, rendered plots, and so on.
+As long as this field is presetn you may add any additional data to that object that you want. This could include information like detailed test results, stdout/stderr, compiler errors, rendered plots, and so on.
```

Comment:
```suggestion
As long as this field is present you may add any additional data to that object that you want. This could include information like detailed test results, stdout/stderr, compiler errors, rendered plots, and so on.
```